Backpack/src/androidTest/java/net/skyscanner/backpack/dialog/BpkDialogTest.kt

```diff
@@ -25,7 +25,7 @@ import org.junit.runner.RunWith
 class BpkDialogTest {
 
   private lateinit var handler: Handler
-  private lateinit var mActivity: Activity
+  private lateinit var mActivity: AppCompatActivity
```

Comment:
Can we maybe drop the `m` prefix?

Reply:
I can take this up as a separate PR and do it in the complete codebase.